plugins/ScheduledReports/API.php

```diff
@@ -450,6 +450,9 @@ public function generateReport($idReport, $date, $language = false, $outputType
         $description = str_replace(array("\r", "\n"), ' ', $report['description']);
 
         list($reportSubject, $reportTitle) = self::getReportSubjectAndReportTitle(Site::getNameFor($idSite), $report['reports']);
+        
+        if(is_array($segment) && strlen($segment['name'])) $reportTitle .= " - ".$segment['name'];
```

Comment:
could you use the same coding style eg. add the `{` and put the $reportTitle on a new line? thanks!